src/+otp/+constrainedpendulum/ConstrainedPendulumProblem.m

```diff
@@ -0,0 +1,35 @@
+classdef ConstrainedPendulumProblem < otp.Problem
```

Comment:
Perhaps just `PendulumDAEProblem`.

Reply:
This class ought to override `internalIndex2label`

Reply:
Agree with both